source/manual/monitor-sidekiq-workers.html.md

```diff
@@ -22,6 +22,20 @@ Then visit [http://127.0.0.1:9000](http://127.0.0.1:9000) to see a list of [Side
 
 You can also monitor Sidekiq queue lengths using [this Grafana dashboard](https://grafana.publishing.service.gov.uk/dashboard/file/sidekiq.json).
 
+Or to view your local Sidekiq queue, within your vm go to govuk-puppet/development-vm and run:
+
+```bash
+$ vagrant ssh  -- -CNL 3091:localhost:3091
+```
+
+Then in the [sidekiq-monitoring app](https://github.com/alphagov/sidekiq-monitoring), run:
+
+```bash
+./bin/foreman run rummager
```

Comment:
Thanks, updated now! I've made a couple of additional changes to make the instructions non-app specific.